Formula/nnn.rb

```diff
@@ -39,9 +39,8 @@ def install
 
     (testpath/"testdir").mkdir
     cd testpath/"testdir" do
-      PTY.spawn(bin/"nnn") do |r, w, _pid|
-        w.write "q"
-        assert_match "~/testdir", r.read
+      PTY.spawn(bin/"nnn", "-V") do |r, _w, _pid|
+        assert_match "4.3", r.read
```

Comment:
This will break on the next update, but also it's not testing much. What is actually going wrong in the old test?

Reply:
I plan to update the version in the future updates as well. The older check has broken once earlier as well as it's an area prone to change.

What do you mean by _doesn't test much_? What is that important for a packager?

Reply:
Also, she version and sha256 needs to be updated on every release as well, isn't it? This also follows suit.

Reply:
> What do you mean by doesn't test much? What is that important for a packager?

Because we need to know that the package works. Most version code never executes anything of the main program so you're not testing if the actual program works.

Reply:
That's is testing the binary not the packaging procedure. I am not sure that from a packager point of view you need to test anything other than the a binary executes or not, and that's a version test in native binaries. It's not necessary to test any other or all paths for a package manager.

Reply:
Experience tells us that that is not the case, so we are going to continue keeping our own standards to make sure that users get the quality they expect from brew. See https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula to see what would be an acceptable test.

Reply:
Also @SMillerDev, I took that check out because I was spending more time in debugging the same issue again and again. Now I see I am spending time discussing non-issues (from a packager point of view) with you. Yesterday, I waited for a while for your reply before taking this route and you didn't bother. Now you came back again with cosmetic issues. `nnn` is packaged by tons of other packagers and not a single one requires this kind of a test. If your packaging process is not able to generate single native binaries from C code I am sure you will come to know. Please don't push it on individual packages to take it too far.

Also, I took the version check route because I saw another package do the same. If it appears to me that I have to spend a ridiculous amount of time to submit an update on a packager I would have to stop doing that.